qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -124,21 +128,29 @@ def pprint_programs(self, refresh: bool = False, detailed: bool = False) -> None
                 print(f"Name: {prog.name}")
                 print(f"Description: {prog.description}")
 
-    def programs(self, refresh: bool = False) -> List[RuntimeProgram]:
+    def programs(self, refresh: bool = False,
+                 limit: int = None, skip: int = None) -> List[RuntimeProgram]:
```

Comment:
Why is the limit "default" is `None` when it's really 20?

Reply:
Do we want to refresh every time a user inputs a limit or skip - or just refresh only when `refresh=True`? If we want to refresh when skip/limit are inputed is there a different way to do that?